Xamarin.Forms.Core/Application.cs

```diff
@@ -31,9 +29,6 @@ public class Application : Element, IResourcesProvider, IApplicationController,
 
 		public Application()
 		{
-			var f = false;
```

Comment:
As silly as this looks, this code serves a purpose. Please do not remove it. Thank you!

Reply:
Ok, no problem.

The code is here from the first commit on github, so it don't provide any hint about its meaning but as the variable is local to the function and I do not see any way to change it.
I thought the change was without impact, but since it's not directly related to this PR I leave that to a future "house keeping" commit.

Reply:
Yep, it definitely deserves a comment. This code is here to prevent the iOS linker from completely removing the Xamarin.Forms.Platform DLL.